adoc/admin-security-ldap.adoc

```diff
@@ -29,7 +28,6 @@ automatically self-signed certificate and key.
 docker run -d \
 	-p 389:389 \
```

Comment:
Just had realized this in the entrypoint.sh:

[slapd]
ServerPort=3389
;SecurePort=3636

--> I had the problem that ldap on 389 was not reachable - so I see that run vs. entrypoint.sh does not match here, too.

Could you either change the run or the entrypoint.sh so that this matches?